meta-lmp-base/recipes-bsp/u-boot/u-boot-ostree-scr-fit/boot-common.cmd.in

```diff
@@ -63,6 +83,8 @@ if test "${fiovb.is_secondary_boot}" = "0"; then
 		if test ! $? -eq 0; then fiovb write_pvalue bootupgrade_available 0; fi
 		fiovb read_pvalue bootfirmware_version 128
 		if test ! $? -eq 0; then run bootcmd_bootenv; fiovb write_pvalue bootfirmware_version "${bootfirmware_version}"; fi
+		fiovb read_pvalue debug 4
+		if test ! $? -eq 0; then fiovb write_pvalue debug 0; fi
```

Comment:
This will also need another optee-os change, similar to https://github.com/foundriesio/optee_os/commit/d3aabed77580dbb49ca766783f41af5881e538b9.

Reply:
done https://github.com/foundriesio/optee_os/pull/65

Reply:
Great, just include the optee bump as part of this pr and we should be good to land.

Reply:
done